opentelemetry/OpenTelemetry-Spans.md

```diff
@@ -11,7 +11,8 @@ Here is the process:
 1. Take all OpenTelemetry span attributes, and map them directly over to New Relic span attributes.
 1. If the OpenTelemetry span has a `status` that is not `OK`, and there is a status description available,
 add an `"error.message"` attribute that contains the status description.
-1. If there is a Resource associated with the Span, take all of the Resource's labels and add them as
-span attributes.
+1. If there is a Resource associated with the Span, take all of the Resource's attributes and add them as
+span attributes. One exception is the `service.name` attribute, if the exporter has been configured with a
+service name, then this name should take precedence over the `service.name` attribute on the resource.
```

Comment:
We will want to include the case where the `service.name` attribute is also set on the span directly. It would seem that this should take the lower precedence than a resource give it would be defined by the instrumentor instead of the operator of a service who would define it on the resource (or exporter) and they will generally not have as much context.

The precedence of the exporter though over the resource will lead to the situation where there are multiple services using the same exporter, as is the case for the OpenTelemetry collector or a server that serves multiple functions, and they will be incorrectly unified into a single service.

With this in mind, should the precedence order be this instead?

1. Resource attribute
2. Span attribute
3. Exporter service name

In this situation the exporter service name would act more as a fallback to ensure that the telemetry would at least show up in New Relic's system as a service if nothing is provided, but the specificity of the service itself would rely on the semantic conventions of OpenTelemetry.